app-config.yaml

```diff
@@ -94,7 +94,7 @@ techdocs:
   generators:
     techdocs: 'docker' # Alternatives - 'local'
   publisher:
-    type: 'local' # Alternatives - 'googleGcs' or 'awsS3' or 'azureBlobStorage'. Read documentation for using alternatives.
+    type: 'local' # Alternatives - 'googleGcs' or 'awsS3' or 'azureBlobStorage' or 'openStackSwift' Read documentation for using alternatives.
```

Comment:
done that

Reply:
I'll just add, this will keep growing and growing. I wonder at some point if we should change this to link to point here or something?

https://backstage.io/docs/features/techdocs/techdocs-overview#platforms-supported

Reply:
That also reminds me - this should totally have an "Integrations" entry in the docs as well! So people can see that it exists and how to configure it